requirements.in

```diff
@@ -1,5 +1,5 @@
-git+ssh://git@github.com/codecov/shared.git@bef9260a4b6218b5ce4b5b9152a71d7e6d63a954#egg=shared
-git+ssh://git@github.com/codecov/opentelem-python.git@v0.0.4a1#egg=codecovopentelem
+https://github.com/codecov/shared/archive/bef9260a4b6218b5ce4b5b9152a71d7e6d63a954#egg=shared
```

Comment:
is this how we're gonna update shared from now on?

Reply:
Short answer yes. Its the same setup. Just update the sha in the url.

@matt-codecov has some larger scope changes for shared in the works. However for now, moving to https allows us to not use the ssh key mounting for pip + docker. That was not necessary since becoming source available.